sklearn_crfsuite/_fileresource.py

```diff
@@ -35,6 +35,7 @@ def cleanup(self):
         if self.name is not None:
             try:
                 os.unlink(self.name)
+                os.close(self.fd)
```

Comment:
Likely you're right, but this order raises questions and requires knowledge to understand what's going on, while the more straightforward order doesn't => it makes code easier to read, so let's reverse it :) 

Do you prefer me to change the order after merging the PR, or are you up to changing it in the PR?